specification/cognitiveservices/data-plane/Personalizer/preview/v2022-09-01-preview/Personalizer.json

```diff
@@ -0,0 +1,2847 @@
+					"200": {
+						"description": "Success",
+						"schema": {
+							"$ref": "#/definitions/FeatureImportance"
+						}
+					},
+					"default": {
+						"description": "Feature importance not found.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"Successful FeatureImportances_Get request": {
+						"$ref": "./examples/FeatureImportances_Get.json"
+					}
+				}
+			}
+		},
+		"/featureImportances": {
+			"get": {
+				"tags": [
+					"FeatureImportances20220901Preview"
+				],
+				"summary": "List Feature Importances.",
+				"description": "List of all Feature Importances.",
+				"operationId": "FeatureImportances_List",
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"name": "api-version",
+						"in": "query",
+						"required": true,
+						"type": "string",
+						"description": "Client API version."
+					},
+					{
+						"in": "query",
+						"name": "orderby",
+						"description": "The ordering. Defaults to creation time in descending order.",
+						"type": "string",
+						"default": "creationTime desc"
+					},
+					{
+						"in": "query",
+						"name": "top",
+						"description": "The maximum number of resources to return from the collection. Defaults to maximum value of integer.",
+						"type": "integer",
+						"format": "int32",
+						"default": 2147483647
+					},
+					{
+						"in": "query",
+						"name": "skip",
+						"description": "An offset into the collection of the first resource to be returned. Defaults to 0.",
+						"type": "integer",
+						"format": "int32",
+						"default": 0
+					},
+					{
+						"in": "query",
+						"name": "maxpagesize",
+						"description": "The maximum number of resources to include in a single response. Defaults to 100.",
+						"type": "integer",
+						"format": "int32",
+						"default": 100
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Success",
+						"schema": {
+							"$ref": "#/definitions/FeatureImportances"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"Successful FeatureImportances_List request": {
+						"$ref": "./examples/FeatureImportances_List.json"
+					}
+				}
+			},
+			"post": {
+				"tags": [
+					"FeatureImportances20220901Preview"
+				],
+				"summary": "Create Feature Importance.",
+				"description": "Submit a new Feature Importance job.",
+				"operationId": "FeatureImportances_Create",
+				"consumes": [
+					"application/json"
+				],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"name": "api-version",
+						"in": "query",
+						"required": true,
+						"type": "string",
+						"description": "Client API version."
+					},
+					{
+						"in": "body",
+						"name": "featureImportance",
+						"description": "The Feature Importance job definition.",
+						"schema": {
+							"$ref": "#/definitions/FeatureImportanceContract"
+						}
+					}
```

Comment:
This body parameter should probably be marked as required, since the body schema has required properties.

Reply:
Made required